docs/Tutorials/Hello_MatMul.md

```diff
@@ -70,21 +71,21 @@ nest = acc.Nest(shape=(M, N, K))
 i, j, k = nest.get_indices()
 ```
 
-Next we define the logic of each iteration of the loop nest:
+Next, we define the logic for each iteration of the loop nest:
 ```python
 # Define the loop nest logic
 @nest.iteration_logic
 def _():
     C[i, j] += A[i, k] * B[k, j]
 ```
 
-We have finished defining the logic of MatMul, and let's define the schedule which controls how the logic is executed. To do this, we first create the schedule from the nest:
+We have finished defining the logic of MatMul. Let's now define the schedule which controls the execution of logic. For this, we first create the schedule from the nest:
```

Comment:
“Let’s now define the schedule to control the execution of the logic. We create a schedule from the nest:”